src/scss/_case-studies-main.scss

```diff
@@ -0,0 +1,241 @@
+@import "variables.scss";
+
+.case-studies {
+  &__hero {
+    margin-bottom: 58px;
+    .display-3 {
+      width: 100%;
+      line-height: 120%;
+      letter-spacing: -0.04em;
+    }
+    &__lead {
+      width: 100%;
+      font-size: 18px;
+    }
+    p {
+      margin-bottom: 0px !important;
+    }
+  }
+  &__detail {
+    margin-right: 30px;
+  }
+  &__initiatives {
+    padding: 60px 0 0px;
+
+    .section-header {
+      margin-bottom: 30px;
+    }
+
+    .d-flex {
+      flex-wrap: wrap;
+      justify-content: space-between;
+    }
+    .card {
+      h5 {
+        display: flex;
+        align-items: center;
+
+        span {
+          font-size: 14px;
+        }
+
+        img {
+          margin: 0 5px 3px 10px;
+        }
+      }
+    }
+  }
+  &__mmr {
+    padding: 40px 0 58px;
+    background: $ash;
+
+    p {
+      max-width: 580px;
+      width: 100%;
+      margin: 0 auto 20px;
+      text-align: center;
+      color: $black;
+    }
+  }
+  &__posts {
+    padding: 100px 0 0;
+
+    .section-header {
+      margin-bottom: 7px;
+
+      img {
+        vertical-align: unset;
+      }
+    }
+
+    .d-flex {
+      flex-wrap: wrap;
+      justify-content: space-between;
+      margin-top: 43px;
+    }
+  }
+  .cases {
+    color: #161616;
+  }
+  .card {
+    padding: 0px;
+    margin-bottom: 30px;
+    width: calc((100% - 80px) / 3);
+    border-radius: 5px;
+    .finn {
+      background-image: url("../casestudiesmain/finnauto.webp");
+    }
+    .locus {
+      background-image: url("../casestudiesmain/locus.webp");
+    }
+    .schwarz {
+      background-image: url("../casestudiesmain/schwarz.webp");
+    }
+    .password {
+      background-image: url("../casestudiesmain/1pass.webp");
```

Comment:
This images directory should be `case-studies`

Reply:
Done :) Now both case studies main index and finn.auto index are on the same folder (case-studies)